lib/core/src/lib/card-view/models/card-view-dateitem.model.ts

```diff
@@ -63,6 +63,6 @@ export class CardViewDateItemModel extends CardViewBaseItemModel<DateItemType> i
     }
 
     private prepareDate(date: Date): Date {
-        return this.type === 'date' ? DateFnsUtils.forceLocal(date) : date;
+        return this.type === 'date' && date instanceof Date ? DateFnsUtils.forceLocal(date) : date;
```

Comment:
```
return this.type === 'date'
      ? DateFnsUtils.forceLocal(date instanceof Date ? date : new Date(date))
      : date;
```
in this way we don't change behavior when `this.type === 'date'`
wdyt?

Reply:
yes, thanks! :)